src/guides/v2.3/config-guide/prod/config-reference-systemxml.md

```diff
@@ -0,0 +1,359 @@
+| `header_css`     | Neither used nor evaluated at the time of this writing.                                                               | `string`            |
+| `resource`       | References an ACL resource to provide permission settings for this section.                                           | `typeAclResourceId` |
+| `group`          | Define one or more subgroups.                                                                                        | `typeGroup`         |
+| `frontend_model` | Specifies a different frontend model to change the rendering and modify the output.                                   | `typeModel`         |
+| `include`        | Used to include additional `system_include.xsd` compatible files. Usually used to structure large `system.xml` files. | `includeType`       |
+
+### Example: Create a new section and assign it to a tab
+
+The following code snippet demonstrates the basic usage of creating a new section.
+
+```xml
+<?xml version="1.0" ?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="A_UNIQUE_ID" translate="label" class="a-custom-css-class-to-style-this-tab" sortOrder="10">
+            <label>A meaningful label</label>
+        </tab>
+
+        <section id="A_UNIQUE_SECTION_ID" showInDefault="1" showInWebsite="0" showInStore="1" sortOrder="10" translate="label">
+            <label>A meaningful section label</label>
+            <tab>A_UNIQUE_ID</tab>
+            <resource>VENDOR_MODULE::path_to_the_acl_resource</resource>
+        </section>
+    </system>
+</config>
+```
+
+The section described above defines the ID `A_UNIQUE_SECTION_ID`, is visible in the default config view and in a store context. The `label`-node is translatable. The section is associated to the tab with the ID `A_UNIQUE_ID`. The section can only be accessed by users that have the permissions defined in the ACL `VENDOR_MODULE::path_to_the_acl_resource`.
+
+## Groups
+
+The `<group>`-Tag is used to group fields together.
+
+### Group attribute reference
+
+A `<group>`-Tag can have the following attributes:
+
+| Attribute       | Description                                                                                                                                         | Type     | Required |
+|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|----------|----------|
+| `id`            | Defines the identifier that is used referencing the group.                                                                                          | `typeId` | required |
+| `translate`     | Defines the field that should be translatable. Provide `label` to make the label translatable.                                                      | `string` | optional |
```

Comment:
For `group`, and `field` tags there's a possibility to translate a group of fields like "label comment" but not the single field as for `tab` and `section`

```suggestion
| `translate`     | Defines fields that should be translatable. Provide `label` to make the label translatable.                                                      | `string` | optional |
```

Reply:
I didn't know that. I Will research it and describe it accordingly.